2996_reflection/reflection.md

```diff
@@ -3021,6 +3021,28 @@ Change the grammar for `$operator-or-punctuator$` in paragraph 1 of [lex.operato
 ```
 :::
 
+### [basic.def.odr]{.sref} One-definition rule {-}
+
+Preprend before paragraph 14 of [basic.def.odr]{.sref}:
+
+::: std
+
+::: addu
+
+[14pre]{.pnum} If a class `C` is defined in a translation with a call to `std::meta::define_class`, every definition of that class shall be the result of a call to `std::meta::define_class` such that its respective members are equal in number and have respectively the same types, alignments, [[no_unique_address]] attributes (if any), bit-field widths (if any), and specified names (if any).
```

Comment:
translation *unit*.

Also we don't have `[[no_unique_address]]` support yet?

Reply:
I added that.  (Again, I have the distinct recollection that I at least added it to the options type before.)